### PR TITLE
Exchange Kovan for Laika deployment comment

### PR DESCRIPTION
**Pull request by weilbith**
Close: #230

@berndbohmeier I don't know if the second sentence of the test Ether still makes sense as it is. Please decide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ import { TLNetwork } from 'trustlines-clientlib'
 
 Use the following configuration to connect to the currently deployed test setup.
 
-**NOTE: The [trustlines-network contracts](https://github.com/trustlines-network/contracts) are deployed on the Kovan testnet. Some Kovan Test ETH is therefore required to interact with the contracts in this setup.**
+**NOTE: The [trustlines-network contracts](https://github.com/trustlines-network/contracts) are deployed on the (_Trustlines_) Laika testnet. Some Laika test `TLC` is therefore required to interact with the contracts in this setup.**
```
Comment (berndbohmeier) [inline README.md]: Lets add a link to the Laika blockexplorer. The sentence about the tlc is still right.

Comment (weilbith) [reply README.md]: Good idea. I have done so. :heavy_check_mark:

File: README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ import { TLNetwork } from 'trustlines-clientlib'
 
 Use the following configuration to connect to the currently deployed test setup.
 
-**NOTE: The [trustlines-network contracts](https://github.com/trustlines-network/contracts) are deployed on the Kovan testnet. Some Kovan Test ETH is therefore required to interact with the contracts in this setup.**
+**NOTE: The [trustlines-network contracts](https://github.com/trustlines-network/contracts) are deployed on the (_Trustlines_) [Laika testnet](https://blockexplorer.trustlines.foundation/). Some Laika test `TLC` is therefore required to interact with the contracts in this setup.**
```
Comment (berndbohmeier) [inline README.md]: This is the wrong link..

Comment (berndbohmeier) [reply README.md]: I'll change it.

**Pull request merged**